genisys/templates/preseed.cfg.jinja2

```diff
@@ -95,9 +95,13 @@ in-target chmod 700 /root/.ssh/
 {% endif %}
 
 # copy firstboot.service file to client 
-d-i preseed/late_command string cp -r /ftp/first-boot/genisys-firstboot.service /etc/systemd/system/
+d-i preseed/late_command string ftp [{{ ftp['username'] }}@]{{ ftp['ip'] }}[:{{ ftp['port'] }}]/ftp/first-boot/genisys-firstboot.service /etc/systemd/system/
```

Comment:
My bad I didn't clarify. When I said that the format was `ftp [<username>@]<ip>[:<port>]/path/on/server /path/on/client` the square brackets indicated that the section was optional. There should not be literal square brackets in the command (unless its ipv6 but thats a different issue).

Also, I don't know if we are specifying an ftp user. Check the config file and with @BBergle since he wrote the FTP config.

Since there's a lot of variation, it might be worth bulding the ftp command by pieces in preseed.py then just templating it in as a single string. Its up to you, the main thing is we don't want the program crashing because the template is trying to grab a configuration option that doesn't exist. Check the tests action on the latest commit to see an example.